app/controllers/ips_controller.rb

```diff
@@ -2,12 +2,13 @@ class IpsController < ApplicationController
   before_action :authorise_manage_locations, only: %i(create new)
 
   def new
-    @ip = Ip.new
+    selected_location = Location.find_by(id: params[:location])
```

Comment:
Would it be better to use:
```
current_organisation.locations.find(params.fetch(:location))
```

To ensure that the found location belongs to the organisation of the current user?

Reply:
If the found location isn't in the organisation, the location selector will just default to the first entry. Not too bad. Would have to hack the URL to cause that behaviour, too.

Good spot though - I think people can use this controller to add IPs to locations that aren't in their organisation.

I think this is pre-existing though - I'd like to address it in a different PR so I can get this one merged, if that's acceptable to you?